### PR TITLE
Clean up and optimization in cdc tests

### PR DESCRIPTION
**Pull request by spillai14**
## Changes proposed in this pull request

The changes mainly include cleanup and formatting changes.

## Patch testing

(Fill in the details about how this patch was tested)

## ReleaseNotes.txt changes

(Does this change require an entry in ReleaseNotes.txt? If yes, has it been added to it?)

## Other PRs 

(Does this change require changes in other projects- store, spark, spark-jobserver, aqp? Add the links of PR of the other subprojects that are related to this change)

### CODE REVIEW

File: dtests/src/resources/scripts/cdcConnector/createAndLoadCDCMixedTables.sql [hunk 1]
```diff
@@ -218,6 +218,7 @@ CREATE EXTERNAL TABLE IF NOT EXISTS  staging_UM_SERVICE
 CREATE TABLE IF NOT EXISTS UM_SERVICE USING column OPTIONS(partition_by 'UM_RVW_ID',redundancy '1',buckets '32',key_columns 'CLIENT_ID,UM_RVW_ID,UM_SERV_ID ' ) AS (SELECT * FROM staging_UM_SERVICE);
 
 DROP TABLE IF EXISTS INQUIRY;
+DROP TABLE IF EXISTS staging_INQUIRY;
 CREATE EXTERNAL TABLE IF NOT EXISTS  staging_INQUIRY
     USING com.databricks.spark.csv OPTIONS (path '/export/shared/QA_DATA/TMG_Data/TMG_RowTable_Data/INQUIRY.dat', header 'true', inferSchema 'true', nullValue 'NULL', maxCharsPerColumn '4096');
```
Comment (swatisawant) [inline dtests/src/resources/scripts/cdcConnector/createAndLoadCDCMixedTables.sql]: Parametrize the variables in sql script and pass them through bt or conf file at runtime.

File: dtests/src/resources/scripts/cdcConnector/createAndLoadFewTables.sql [hunk 2]
```diff
@@ -19,30 +19,62 @@ CREATE EXTERNAL TABLE staging_BANK
     USING com.databricks.spark.csv OPTIONS (path '/export/shared/QA_DATA/TMG_Data_20G/BANK.dat', header 'true', inferSchema 'true', nullValue 'NULL', maxCharsPerColumn '4096');
```
Comment (swatisawant) [inline dtests/src/resources/scripts/cdcConnector/createAndLoadFewTables.sql]: Parametrize the variables in sql script and pass them through bt or conf file at runtime.

File: dtests/src/resources/scripts/cdcConnector/createAndLoadFewTables.sql [hunk 3]
```diff
@@ -19,30 +19,62 @@ CREATE EXTERNAL TABLE staging_BANK
...
 CREATE TABLE BANK USING column OPTIONS(partition_by 'BNK_ORG_ID', buckets '32',key_columns 'CLIENT_ID,BNK_ORG_ID,BNK_ID ' ) AS (SELECT * FROM staging_BANK);
 
+DROP TABLE IF EXISTS BENEFIT_PACKAGE;
+DROP TABLE IF EXISTS staging_BENEFIT_PACKAGE;
+CREATE EXTERNAL TABLE IF NOT EXISTS  staging_BENEFIT_PACKAGE
+    USING com.databricks.spark.csv OPTIONS (path '/export/shared/QA_DATA/TMG_Data_20G/BENEFIT_PACKAGE.dat', header 'true', inferSchema 'true', nullValue 'NULL', maxCharsPerColumn '4096');
```
Comment (swatisawant) [inline dtests/src/resources/scripts/cdcConnector/createAndLoadFewTables.sql]: Parametrize the variables in sql script and pass them through bt or conf file at runtime.

File: dtests/src/resources/scripts/cdcConnector/createAndLoadFewTables.sql [hunk 4]
```diff
@@ -19,30 +19,62 @@ CREATE EXTERNAL TABLE staging_BANK
...
     USING com.databricks.spark.csv OPTIONS (path '/export/shared/QA_DATA/TMG_Data_20G/BANK_ACCOUNT.dat', header 'true', inferSchema 'true', nullValue 'NULL', maxCharsPerColumn '4096');
-CREATE TABLE BANK_ACCOUNT USING column OPTIONS(partition_by 'BNK_ORG_ID',buckets '32',key_columns 'CLIENT_ID,BNK_ORG_ID,BNK_ID,BNK_ACCT_ID ' ) AS (SELECT * FROM staging_BANK_ACCOUNT);
+ CREATE TABLE IF NOT EXISTS BANK_ACCOUNT USING column OPTIONS(partition_by 'BNK_ORG_ID',redundancy '1', buckets '32',key_columns 'CLIENT_ID,BNK_ORG_ID,BNK_ID,BNK_ACCT_ID' ) AS (SELECT * FROM staging_BANK_ACCOUNT);
 
 DROP TABLE IF EXISTS BENEFIT_GROUP_NAME;
 DROP TABLE IF EXISTS staging_BENEFIT_GROUP_NAME;
 ----- CREATE TEMPORARY STAGING TABLE TO LOAD CSV FORMATTED DATA -----
-CREATE EXTERNAL TABLE staging_BENEFIT_GROUP_NAME
+CREATE EXTERNAL TABLE IF NOT EXISTS  staging_BENEFIT_GROUP_NAME
     USING com.databricks.spark.csv OPTIONS (path '/export/shared/QA_DATA/TMG_Data_20G/BENEFIT_GROUP_NAME.dat', header 'true', inferSchema 'true', nullValue 'NULL', maxCharsPerColumn '4096');
```
Comment (swatisawant) [inline dtests/src/resources/scripts/cdcConnector/createAndLoadFewTables.sql]: Parametrize the variables in sql script and pass them through bt or conf file at runtime. Please check for all the sql scripts.

File: dtests/src/test/java/io/snappydata/hydra/cdcConnector/cdcConnectorLongRunningHAWithMultiSqlServer.conf [hunk 5]
```diff
@@ -15,27 +15,27 @@ INCLUDE $JTESTS/io/snappydata/hydra/cdcConnector/snappyCustomizedHA.inc;
...
             maxThreads = 1
             ;
 
 TASK        taskClass  = io.snappydata.hydra.cluster.SnappyTest taskMethod  = HydraTask_executeSparkJob
             io.snappydata.hydra.cluster.SnappyPrms-sparkJobClassNames = io.snappydata.app.JavaCdcStreamingApp
-            io.snappydata.hydra.cluster.SnappyPrms-sparkSubmitExtraPrms = " --total-executor-cores=&{coresPerStrmApp} --conf spark.scheduler.mode=FAIR --conf spark.sql.defaultSizeInBytes=1000 --conf spark.driver.memory=10g --conf spark.executor.memory=4g --conf snappydata.store.memory-size=12g --conf spark.locality.wait=30 --conf spark.local.dir=/nfs/users/spillai/tmp1 --jars ${connectorJar},${driverJar},${testJar} "
-            io.snappydata.hydra.cluster.SnappyPrms-userAppArgs = "${connectionPropFile2} ${sourceDestFile2} "
+            io.snappydata.hydra.cluster.SnappyPrms-sparkSubmitExtraPrms = " --total-executor-cores=6 --conf spark.scheduler.mode=FAIR --conf spark.sql.defaultSizeInBytes=1000 --conf spark.driver.memory=10g --conf spark.executor.memory=4g --conf snappydata.store.memory-size=12g --conf spark.locality.wait=30 --conf spark.local.dir=/nfs/users/spillai/tmp1 --jars ${connectorJar},${driverJar},${testJar} "
```
Comment (swatisawant) [inline dtests/src/test/java/io/snappydata/hydra/cdcConnector/cdcConnectorLongRunningHAWithMultiSqlServer.conf]: Formatting changes required for all bts and confs

File: dtests/src/resources/scripts/cdcConnector/createAndLoadCDCMixedTables.sql (hunk 1 above)
Comment (spillai14) [reply dtests/src/resources/scripts/cdcConnector/createAndLoadCDCMixedTables.sql]: Done

File: dtests/src/resources/scripts/cdcConnector/createAndLoadFewTables.sql (hunk 2 above)
Comment (spillai14) [reply dtests/src/resources/scripts/cdcConnector/createAndLoadFewTables.sql]: Done

File: dtests/src/resources/scripts/cdcConnector/createAndLoadFewTables.sql (hunk 3 above)
Comment (spillai14) [reply dtests/src/resources/scripts/cdcConnector/createAndLoadFewTables.sql]: Done

File: dtests/src/resources/scripts/cdcConnector/createAndLoadFewTables.sql (hunk 4 above)
Comment (spillai14) [reply dtests/src/resources/scripts/cdcConnector/createAndLoadFewTables.sql]: Done

File: dtests/src/test/java/io/snappydata/hydra/cdcConnector/cdcConnectorLongRunningHAWithMultiSqlServer.conf (hunk 5 above)
Comment (spillai14) [reply dtests/src/test/java/io/snappydata/hydra/cdcConnector/cdcConnectorLongRunningHAWithMultiSqlServer.conf]: Done

File: dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java [hunk 6]
```diff
@@ -2277,25 +2277,9 @@ public void executeSparkJob(Vector jobClassNames, String logFileName) {
             SnappyBB.getBB().getSharedMap().put("START_RANGE_APP2", finalEnd2 + 1);
             SnappyBB.getBB().getSharedMap().put("END_RANGE_APP2", finalEnd2 + 100);
           } else if (appName.equals("CDCIngestionApp1")) {
-            userAppArgs = userAppArgs + " " + finalStart + " " + finalEnd;
+            userAppArgs = finalStart + " " + finalEnd + " " + userAppArgs;
```
Comment (sonalsagarwal) [inline dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java]: The CDC code can be moved to a method and the method call be called here.

Comment (spillai14) [reply dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java]: Done

**Pull request merged**